README.md

```diff
@@ -114,37 +113,33 @@ available---for instance, at the time of this writing, Ubuntu 16.04 only has
 mlpack 2.0.1 available.  Options include upgrading your Ubuntu version, finding
 a PPA or other non-official sources, or installing with a manual build.
 
-There are some other useful pages to consult in addition to this section:
+There are some useful pages to consult in addition to this section:
 
   - [Building mlpack From Source](http://www.mlpack.org/docs/mlpack-git/doxygen/build.html)
   - [Building mlpack From Source on Windows](http://www.mlpack.org/docs/mlpack-git/doxygen/build_windows.html)
 
 mlpack uses CMake as a build system and allows several flexible build
-configuration options. One can consult any of numerous CMake tutorials for
-further documentation, but this tutorial should be enough to get mlpack built
-and installed.
+configuration options. The users can consult any of the CMake tutorials for
+further documentation, but this tutorial can get mlpack built and installed.
 
 First, unpack the mlpack source and change into the unpacked directory.  Here we
 use mlpack-x.y.z where x.y.z is the version.
 
     $ tar -xzf mlpack-x.y.z.tar.gz
     $ cd mlpack-x.y.z
 
-Then, make a build directory.  The directory can have any name, not just
-'build', but 'build' is sufficient.
+Then, make a build directory.  The directory can have any name, but 'build' is sufficient.
```

Comment:
Fine by me, but can you be sure to wrap this line to 80 characters in accorance with the style guide please?